template/php7/readme.md

```diff
@@ -9,13 +9,13 @@ Templates are built using the latest minor version of the major release.
 ## Usage:
 
 ```shell
-faas-cli new my-function-php7 --lang php7
+faas-cli new my-function-php --lang php7
 ```
 
-You will find in the newly created directories `my-function-in-php5` and `my-function-in-php7`:
+You will find in the newly created directory `my-function-in-php`:
```

Comment:
Directory is still wrong here I'm afraid, `-in-`. Would really prefer we drop the `-php` in the function name, when consuming a function one of the points is that the user shouldn't have to know or care what language its implemented in. Ie you could replace `my-function` with a Ruby, Python or whatever implementation and you don't have to change any code using that function.

Reply:
Makes sense, I'll make the change.